gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -4973,6 +4973,13 @@ gnc_plugin_page_register_cmd_scrub_current (GtkAction* action,
     LEAVE (" ");
 }
 
+static void cancel_scrub (GtkWidget *window)
+{
+    abort_scrub = gnc_verify_dialog
+        (GTK_WINDOW (window), FALSE,
+         _("A scrubbing operation is currently pending, do you want to abort it?"));
```

Comment:
[s/scrubbing/Check and Repair/, s/pending/running/](https://code.gnucash.org/logs/2020/09/16.html#T15:37:03)